addons/binding/org.openhab.binding.atlona/src/main/java/org/openhab/binding/atlona/internal/net/SocketChannelSession.java

```diff
@@ -1,4 +1,4 @@
-package org.openhab.binding.russound.internal.net;
+package org.openhab.binding.atlona.internal.net;
```

Comment:
![screen shot 2017-02-02 at 15 10 01](https://cloud.githubusercontent.com/assets/3244965/22552553/b3267274-e959-11e6-8f3e-2f6e0aa37b3a.png)

This shows that the file is moved (and edited) and not copied. So if I'd merge this PR, the Russound binding will not contain that class anymore.